qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -202,7 +202,7 @@
             type = qemu_guest_agent_snapshot
             gagent_check_type = fsfreeze
             gagent_fs_test_cmd = "echo foo > %s/foo"
-            mountpoint_def = "/tmp"
+            mountpoint_def = "/home"
```

Comment:
Em, then better remove the temporary file at the end of the test.

Reply:
you're right, sounds reasonable.

Reply:
@vivianQizhu 
(1/2) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.qemu_guest_agent.virtio_serial.check_fsfreeze.q35: PASS (212.11 s)